WhaleSpotting/react-app/src/components/ReportSighting.tsx

```diff
@@ -0,0 +1,92 @@
+import "../styles/ReportSighting.scss";
+import React, { useState } from "react";
+
+export default function ReportSighting(): JSX.Element {
+    const [datetime, setDatetime] = useState("");
+    const [location, setLocation] = useState("");
+    const [species, setSpecies] = useState("");
+    const [quantity, setQuantity] = useState("");
+    const [longitude, setLongitude] = useState("");
+    const [latitude, setLatitude] = useState("");
+    const [orcaPod, setOrcaPod] = useState("");
+    const [imageUrl, setImageUrl] = useState("");
+    const [description, setDescription] = useState("");
+
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const sighting = { datetime, location, species, quantity, longitude, latitude, orcaPod, imageUrl, description };
+        console.log(sighting);
+    };
+
+    return (
+        <div className="report-sighting" data-testid="report-sighting">
+            <img className="banner-image" src="https://ssl.tzoo-img.com/images/tzoo.1.0.685904.shutterstock_567677317.jpg" alt="whale" />
+            <div className="container">
+                <div className="title">Report Your Sighting</div>
+                <form action="#" onSubmit={(e)=>handleSubmit(e)}>
+                    <div className="card-component">
+                        <div className="sighting-details">
+                            <div className="input-box">
+                                <label className="details">Sighting Date Time</label>
+                                <input name="datetime" type="date" placeholder="Enter Sighting Date Time" required
+                                    value={datetime}
+                                    onChange={(e) => setDatetime(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label className="details">Location</label>
+                                <input type="text" name="location" placeholder="Confirm your location" required
+                                    value={location}
+                                    onChange={(e) => setLocation(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label className="details">Species</label>
+                                <input type="text" name="species" placeholder="Enter species" required
+                                    value={species}
+                                    onChange={(e) => setSpecies(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label className="details">Quantity</label>
+                                <input type="number" placeholder="Enter quantity"
```

Comment:
Could we either add something that says the required fields are required, or that the optional fields are optional

Reply:
done, let us know what you think, it is enough or would you like a brighter colour for the "(required)" text.